### PR TITLE
Fix type definitions

### LINKED ISSUE #640: Type definition issue with latest release
From #636 

image

It seems the `react-menu` package has some typing references to the original source file. If this is an unrelated problem, I can create another issue.

### PR DESCRIPTION
**Pull request by benoitgrelard**
Fixes #640

~TODO: Still need to check the defs manually to make sure it's all good.~

Update: I've now checked the type defs thoroughly and AFAICT it's all good.

### CODE REVIEW

File: packages/react/collection/src/Collection.tsx [hunk 1]
```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import { useComposedRefs } from '@radix-ui/react-compose-refs';
 import { Slot } from '@radix-ui/react-slot';
 
-type SlotProps = React.ComponentProps<typeof Slot>;
+interface SlotProps extends React.ComponentProps<typeof Slot> {}
```
Comment (jjenzz) [inline packages/react/collection/src/Collection.tsx:5]: Did you mean to leave this one in?

Comment (benoitgrelard) [reply packages/react/collection/src/Collection.tsx:5]: nope!

Comment (benoitgrelard) [reply packages/react/collection/src/Collection.tsx:5]: 👀

**Review by jjenzz — APPROVED** [review]

File: scripts/fix-type-defs-imports.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+console.log('Fix bad imports in d.ts files…');
+
+replaceInFiles({
+  files: 'packages/**/dist/**/*.d.ts',
+  from: /(import\("@radix-ui\/[a-z-]+)[a-zA-Z/"]*"\)/g,
+  to: '$1")',
+})
+  .then(({ countOfMatchesByPaths }) => console.log('Fixed:', countOfMatchesByPaths))
+  .catch((error) => console.error('Error fixing bad imports in d.ts files:', error));
```
Comment (kaisermann) [inline scripts/fix-type-defs-imports.js:11]: beautiful 👌

Comment (benoitgrelard) [reply scripts/fix-type-defs-imports.js:11]: 😀

**Pull request merged**